packages/langium/src/grammar/validation/validator.ts

```diff
@@ -785,6 +789,108 @@ export class LangiumGrammarValidator {
         }
     }
 
+    /** This validation recursively looks at all assignments (and rewriting actions) with '=' as assignment operator and checks,
+     * whether the operator should be '+=' instead. */
+    checkAssignmentOperatorMultiplicities(rule: ast.ParserRule, accept: ValidationAcceptor): void {
+        // for usual parser rules AND for fragments, but not for data type rules!
+        if (!rule.dataType) {
+            this.checkAssignmentOperatorMultiplicitiesLogic([rule.definition], accept);
+        }
+    }
+
+    private checkAssignmentOperatorMultiplicitiesLogic(startNodes: AstNode[], accept: ValidationAcceptor): void {
+        // new map to store usage information of the assignments
+        const map: Map<string, AssignmentUse> = new Map();
+
+        // top-down traversal
+        for (const node of startNodes) {
+            // TODO dürfen mehr als 1 Action auftauchen? ansonsten funktioniert das hier nicht so ganz wie gedacht!
```

Comment:
> dürfen mehr als 1 Action auftauchen? ansonsten funktioniert das hier nicht so ganz wie gedacht!

Try it out. Not sure if this is wanted to have multiple actions in series. If garbage comes out of some experiments it might be good to forbid it using a separate PR (?)...

Reply:
Actually we have an in-series case already with binary operators:

`Expression: AddTerm ({XYZ.left=current} right=AddTerm)*;`

Reply:
Yes, multiple actions are fine. Please don't comment in German though :)

Reply:
Thanks @Lotes for the hint, I again forgot the internal comment after fixing the issue.

Thanks @msujew for the confirmation!

I improved the comments and added a test case with a second action within the same parser rule.